frontend/src/app/user/user.component.html

```diff
@@ -1,43 +1,63 @@
 <div class="jumbotron user-card">
   <div class="container-fluid">
     <div *ngIf="userInfo; else userError" class="container">
-      <div class="row">
-        <div class="col">
+      <p>
+        <a
+        href="#"
+        class="btn btn-light btn-lg rate-button"
+        (click)="open(ratingmodal)">
+          {{ "user.rate-button.text" | translate }}
+        </a>
+      </p>
+      <div class="user-info">
+        <div>
           <img
             class="rounded-circle"
+            alt="profile-picture"
             src="/api/user/pictures/{{userInfo.pictureId}}"
-            width="250"
-            height="250"
+            width="200"
+            height="200"
           />
         </div>
-        <div class="col">
+        <div class="user-details">
           <h1>{{ userInfo.firstName + " " + userInfo.lastName }}</h1>
           <p>{{ userInfo.jobTitle }}</p>
-          <p>
-            <a
-              href="#"
-              class="btn btn-success btn"
-              (click)="open(ratingmodal)"
-              >{{ "user.rate-button.text" | translate }}</a
-            >
-          </p>
         </div>
       </div>
-      <hr class="m4-4" />
 
-      <div *ngIf="!httpError.badgesError; else badgesError" class="row">
-        <div class="row">
-          <ng-container *ngFor="let badge of badges">
-            <div class="col">
-              <img
-                src="../../assets/images/badges/badge-{{ badge.id }}.png"
-                width="100"
-                height="100"
-              />
-            </div>
-          </ng-container>
+      <div class="badge-container" *ngIf="!httpError.badgesError; else badgesError">
+        <ng-container *ngFor="let badge of badges">
+          <img
+            alt="badge"
+            class="badge-image"
+            src="../../assets/images/badges/badge-{{ badge.id }}.png"
+            width="100"
+            height="125"
```

Comment:
Ist die width & height nicht schon in der Klasse badge-image defininert? Können wir das hier löschen?